rag-sql-assistant/src/ui/main.py

```
@@ -0,0 +1,163 @@
+                break
+        
+        # Display header with logos if found
+        col1, col2 = st.columns([6, 1])
+        
+        with col1:
+            if openai_logo_path and llama_cloud_logo_path:
+                # Both logos found
+                st.markdown("""
+                # RAG-SQL Assistant powered by <img src="data:image/png;base64,{}" style="height: 0.9em; vertical-align: middle; padding-left: 5px; padding-right: 5px;"> and <img src="data:image/png;base64,{}" style="height: 0.9em; vertical-align: middle; padding-left: 5px;">
+                """.format(
+                    base64.b64encode(open(openai_logo_path, "rb").read()).decode(),
```

Ruff (0.8.2)

Use a context manager for opening files

(SIM115)